tests/garage/torch/modules/test_mlp_module.py

```diff
@@ -0,0 +1,47 @@
+import pickle
+import unittest
+
+from nose2.tools.params import params
+import numpy as np
+import torch
+import torch.nn as nn
+
+from garage.torch.modules import MLPModule
+
+
+class TestMLPModel(unittest.TestCase):
+    @params((5, 1, (1, )), (5, 1, (2, )), (5, 2, (3, )), (5, 2, (1, 1)),
+            (5, 3, (2, 2)))
+    def test_output_values(self, input_dim, output_dim, hidden_sizes):
+        input_val = torch.ones([1, 5], dtype=torch.float32)
+        module = MLPModule(
+            input_dim=input_dim,
+            output_dim=output_dim,
+            hidden_sizes=hidden_sizes,
+            hidden_w_init=nn.init.ones_,
+            output_w_init=nn.init.ones_)
+        output = module.forward(input_val)
```

Comment:
We don't need to directly call forward here. I think you just need to do `output = module(input_val)`